app/javascript/gobierto_dashboards/webapp/containers/subsidies_shared/Nav.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <nav class="dashboards-home-nav">
+    <ul>
+      <router-link
+        :to="{ name: 'summary' }"
+        :class="{ 'is-active': activeTab === 0 }"
+        tag="li"
+        class="dashboards-home-nav--tab"
+        @click.native="markTabAsActive(0)"
+      >
+        <i class="fas fa-chart-bar" />
+        <i class="far fa-chart-bar" />
```

Comment:
I've seen this. @jorgeatgu, do you remember what happened if we remove the duplicated?

Reply:
Nothing happens, is the same icon but with different weight

far class = font-awesome Regular
fas class = font-awesome Solid